integration_tests/src/main/python/udf_test.py

```diff
@@ -425,3 +426,41 @@ def test_func(spark):
             lambda data: [pd.DataFrame([len(list(data))])], schema="ret:integer")
 
     assert_gpu_and_cpu_are_equal_collect(test_func, conf=arrow_udf_conf)
+
+
+@pytest.mark.skipif(is_before_spark_350(),
```

Comment:
Better to ignore order for result comparison.

Reply:
no need to do that, since there is only 1 partition, the result will not be mess up.